docs/pmm-upgrade/upgrade_docker.md

```diff
@@ -1,62 +1,52 @@
 # Upgrade PMM Server using Docker
 
-??? info "Summary"
+## Before you begin
 
-    !!! summary alert alert-info ""
-        - Stop the running container.
-        - Backup (rename) the container and copy data.
-        - Pull the latest Docker image.
-        - Run it.
-
-    ---
+Before starting the upgrade, complete these preparation steps to ensure you can recover your system if needed and confirm compatibility with the new version:
+{.power-number}
 
-!!! caution alert alert-warning "Important"
-    Downgrades are not possible. To go back to using a previous version you must have created a backup of it before upgrading.
+1. Create a backup before upgrading, as downgrades are not possible. Therefore, reverting to a previous version requires an backup made prior to the upgrade.
 
-!!! hint alert alert-success "Tip"
-    To see what release you are running, use the *PMM Upgrade* panel on the *Home Dashboard*, or run:
+2. Verify your current PMM version: Check your current PMM version by navigating to **PMM Configuration > Updates** or by running the following command. If accessing remotely, replace `localhost` with the appropriate IP address or server name:
 
     ```sh
-    docker exec -it pmm-server \
-    curl -ku admin:admin https://localhost/v1/version
+   docker exec -it pmm-server curl -ku admin:admin https://localhost/v1/version
```

Comment:
I think this will not work within a pmm 3 server container, the default https port changed to 8443.

@BupycHuk could you verify, if that's the case?

Reply:
yes, that's correct.

Reply:
The url should be then like so:

```suggestion
   docker exec -it pmm-server curl -ku admin:admin https://localhost:8443/v1/version
```